features/ftue/impl/src/main/kotlin/io/element/android/features/ftue/impl/FtueFlowNode.kt

```diff
@@ -99,15 +107,15 @@ class FtueFlowNode @AssistedInject constructor(
             NavTarget.SessionVerification -> {
                 val callback = object : FtueSessionVerificationFlowNode.Callback {
                     override fun onDone() {
-                        lifecycleScope.launch { moveToNextStep() }
+                        lifecycleScope.launch { moveToNextStepIfNeeded() }
```

Comment:
Remove `lifecycleScope.launch` here and in the case below? It seems that `moveToNextStepIfNeeded` is handling that (and this is not new).